internal/controller/spectrumxrailpoolconfig_host_flows_controller.go

```
@@ -70,65 +101,449 @@ func NewSpectrumXRailPoolConfigHostFlowsReconciler(
+
+func (r *SpectrumXRailPoolConfigHostFlowsReconciler) generateOVSNetwork(spec *v1alpha1.SpectrumXRailPoolConfigSpec, rt *v1alpha1.RailTopology, addBridge bool, namespace string) *sriovv1.OVSNetwork {
+	ovsNetwork := &sriovv1.OVSNetwork{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      rt.Name,
+			Namespace: namespace,
+		},
+		Spec: sriovv1.OVSNetworkSpec{
+			ResourceName:     rt.Name,
+			InterfaceType:    ovsNetworkInterfaceType,
+			NetworkNamespace: spec.NetworkNamespace,
+			MTU:              uint(rt.MTU),
```

golangci-lint (2.11.3)

[high] G115: integer overflow conversion int -> uint

(gosec)